src/metrics_server/test_integration.action.sh

```diff
@@ -78,22 +82,51 @@ EOF
 # Write the expected responses to temporary files.
 # Ignore the ISO formatted timestamps to ease the comparison.
 cat << EOF > "${CONNECTIONS_EXPECTED_RESPONSE}"
-[{"bytesTransferred":"${BYTES_TRANSFERRED1}","countries":["US","NL"],"serverId":"${SERVER_ID}","userId":"${USER_ID1}"},{"bytesTransferred":"${BYTES_TRANSFERRED2}","countries":["UK"],"serverId":"${SERVER_ID}","userId":"${USER_ID2}"}]
+[
```

Comment:
Please have a case without tunnel time to ensure it's backwards-compatible and doesn't break.

Reply:
Done.